Development.md

```diff
@@ -0,0 +1,100 @@
+# Development guide
+
+This guide will help you set up the `generator` locally, run tests, and use Docker for isolated testing.
+
+## Getting started
+
+1. Fork & Clone the repository:
+
+First fork the repository from github and then clone it,
+
+```bash
+git clone https://github.com/asyncapi/generator.git
+cd generator
```

Comment:
```suggestion
cd apps/generator
```

Reply:
no it should be generator only as, git will make a new directory when cloning

Reply:
right, sorry